CoffeeChat/Cells/CommunityUserTableViewCell.swift

```diff
@@ -0,0 +1,147 @@
+//
+//  CommunityUserTableViewCell.swift
+//  CoffeeChat
+//
+//  Created by Lucy Xu on 10/14/20.
+//  Copyright © 2020 cuappdev. All rights reserved.
+//
+
+import UIKit
+
+class CommunityUserTableViewCell: UITableViewCell {
+
+    // MARK: - Private View Vars
+    private let containerView = UIView()
+    private let informationLabel = UILabel()
+    private var interestsCollectionView: SelfSizingCollectionView!
+    private let profileImageView = UIImageView()
+    private let nameLabel = UILabel()
+
+    // MARK: - Private Data Vars
+    private var interests: [String] = []
+    static let reuseIdentifier = "CommunityUserTableViewCell"
+
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+
+        selectionStyle = .none
+        backgroundColor = .clear
+
+        containerView.backgroundColor = .white
+        containerView.layer.cornerRadius = 12
+        containerView.layer.masksToBounds = true
+        containerView.clipsToBounds = false
+        contentView.addSubview(containerView)
+
+        profileImageView.layer.cornerRadius = 18
+        // TODO: Replace with actual profile image
+        profileImageView.layer.backgroundColor = UIColor.gray.cgColor
+        profileImageView.layer.masksToBounds = true
+        profileImageView.clipsToBounds = true
+        containerView.addSubview(profileImageView)
+
+        nameLabel.font = ._16CircularStdBook
+        nameLabel.textColor = .black
+        containerView.addSubview(nameLabel)
+
+        informationLabel.font = ._10CircularStdBook
+        informationLabel.textColor = .greenGray
+        containerView.addSubview(informationLabel)
+
+        let tagsCollectionViewLayout = LeftAlignedFlowLayout()
+        tagsCollectionViewLayout.minimumInteritemSpacing = 4
+        tagsCollectionViewLayout.minimumLineSpacing = 4
+
+        interestsCollectionView = SelfSizingCollectionView(
```

Comment:
i think this would be cleaner, given that everything else is initialized inline; edit -- and also seeing daniel's later comments about unwrapping etc, i think this method would be better